packages/playwright/src/reporters/html.ts

```diff
@@ -62,11 +63,7 @@ class HtmlReporter implements ReporterV2 {
   private config!: api.FullConfig;
   private suite!: api.Suite;
   private _options: HtmlReporterOptions;
-  private _outputFolder!: string;
-  private _attachmentsBaseURL!: string;
-  private _open: string | undefined;
-  private _port: number | undefined;
-  private _host: string | undefined;
+  private _resolvedOptions!: { outputFolder: string, open: HtmlReportOpenOption, attachmentsBaseURL: string, host: string | undefined, port: number | undefined, title: string | undefined };
```

Comment:
it is not like it reads better now with 1000 cols, extract the type? use ? instead of | undefined while there.